lumen/util.py

```diff
@@ -236,3 +243,49 @@ def extract_refs(spec, ref_type=None):
         return refs
     filtered = [ref for ref in refs if ref[1:].startswith(ref_type)]
     return filtered
+
+
+def catch_and_notify(message=None):
```

Comment:
Great, a decorator is exactly what I was thinking about.

Reply:
Glad to hear!